src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { SlideMenu } from "../SlideMenu/SlideMenu";
+import "./Nav.scss";
+
+export class Nav extends React.Component {
+  render() {
+    return (
+      <nav>
```

Comment:
css 선택자 분리를 위해서 컴포넌트 최상단 태그의 class명을 컴포넌트 명과 같게 해주시면 편리합니다. 
```suggestion
      <nav className="Nav">
```